scripts/docgen/doc_source_map.json

```diff
@@ -1,8 +1,9 @@
 {
-    "build": "src/command_modules/vsts-cli-build/vsts/cli/build/_help.py",
-    "code": "src/command_modules/vsts-cli-code/vsts/cli/code/_help.py",    
-    "team": "src/command_modules/vsts-cli-team/vsts/cli/team/_help.py",
-    "work": "src/command_modules/vsts-cli-work/vsts/cli/work/_help.py",
-    "admin": "src/command_modules/vsts-cli-admin/vsts/cli/admin/_help.py",
-    "package": "src/command_modules/vsts-cli-package/vsts/cli/package/_help.py"
+  "build": "src/command_modules/vsts-cli-build/vsts/cli/build/_help.py",
+  "release": "src/command_modules/vsts-cli-release/vsts/cli/release/_help.py",
```

Comment:
Currently, folder name is **vrelease**. Hence this won't work.

Reply:
I am fixing the folder names. Its working in my local.